examples/Makefile-nrf52840

```diff
@@ -87,14 +87,27 @@ CONFIG_FILE_PATH = $(AbsTopSourceDir)/examples/platforms/nrf52840/
 COMMONCFLAGS                   := \
     -fdata-sections               \
     -ffunction-sections           \
-    -Os                           \
-    -g                            \
     -D$(CONFIG_FILE)              \
     -I$(CONFIG_FILE_PATH)         \
     $(NULL)
 
 include $(dir $(abspath $(lastword $(MAKEFILE_LIST))))/common-switches.mk
 
+ifeq ($(DEBUG),1)
+configure_OPTIONS += --enable-optimizations=no
```

Comment:
This is redundant with [examples/common-switches.mk](https://github.com/openthread/openthread/blob/d57b747c980f7b5a38080db529a23ec387f69bc7/examples/common-switches.mk#L71), which in fact also adds `--enable-debug`.

Reply:
@jwhui ok, thanks for pointing that out.  I'll close this.